src/coordinates.js

```diff
@@ -0,0 +1,108 @@
+/**
+ * Parses Coordinates for pages where the default Wiki Infobox Parser fails.
+ * @param  {Object} infoboxData - raw data object from Wiki Infobox Parser
+ * @return {Object} - formatted object containing coordinates, or null object if none.
+ */
+export function parseCoordinates(infoboxData) {
+	if(infoboxData.coordinates) {
+		// inline coordinate format
+		return parseInfoboxCoords(infoboxData.coordinates);
+	}
+	else if(infoboxData.latd && infoboxData.longd) {
+		// deprecated coordinate format
+		return parseDeprecatedCoords(infoboxData.latd,infoboxData.longd);
+	}else {
+		return {
+			lat: null,
+			lon: null,
+			error: 'No coordinates on page.'
+		};
+	}
+}
+
+// regex to match deprecated coordinates
+const deprecatedCoordinatePattern = /(\d{1,3})\s*\|\s*\w{3,4}m=(\d{1,2})(?:\s*\|\s*\w{3,4}s=)?(\d{1,2})?\s*\|\s*\w+=([NSEW])/;
+
+/**
+ * Parses coordinates which are in Wikipedia Deprecated Format.
+ * @example
+ * parseDeprecatedCoords('00 |latm=47 |lats=59 |latNS=S','100 |longm=39 |longs=58 |longEW=E');
+ * @param  {String} latString - Deprecated coordinate string for latitutde (from latd property)
+ * @param  {String} lonString - Deprecated coordinate string for longitude (from longd property)
+ * @return {Object} - Wiki formatted object containing lat and lon
+ */
+function parseDeprecatedCoords(latString,lonString) {
+	var matches, latitude, longitude;
```

Comment:
Will do -- q: is there a reason to use `let` inside of function? I thought that `let` is good for block scoping (i.e. within `for` loop or similar) but that it doesn't matter in function itself whether I use `let` or `var`. Is this just stylistic to indicate that the variables are only used locally?